### DISCUSSION TITLE
I installed DAMASK for the first time. I do not know what to do next, need help to start using this package. Please help.

### DISCUSSION THREAD

**Discussion by st4rkman** [damask-multiphysics/DAMASK#10] [Q&A] [answered]
I installed DAMASK 3 for my college project, this is the first time me using DAMASK. I don't understand how to use it. The video tutorials on the website are not working and I feel helpless. Kindly guide me.

**Comment by sharanroongta** [comment, marked as answer]
Hi @st4rkman . Sorry for the trouble with respect to broken video links. We are in process of fixing that.
For the time being, you can read the documentation in: https://damask.mpie.de/release/documentation/usage.html

Essentially there are 3 input files requried:
1) Geometry file - specifying the spatial distribution of grains that is referred to by the materialID
2) Material configuration - A file specifying the material/grain properties 
3) Loadcase - The boundary conditions and the solver selection is specified here.

The details of each file is mentioned in the documentation. 
You should also read the damask overview paper - https://damask.mpie.de/files/documentation/RotersEtAl2019.pdf

Let me know if you have further questions. And also please mark this question as answered if there is nothing else to discuss.

**Reply by st4rkman** [reply]
Thank you @sharanroongta this was very helpful. 
I had now run the model and visualized it on para view. The next thing that is on my mind is that how can I learn and understand the material, loading and seed files and how can I manipulate it. As the next consideration is for me to change these parameters according to the need of my project as I have used the default configuration files for current result.
Can you help me with it?

**Reply by sharanroongta** [reply]
Hi! The documentation I had mentioned, and the python library description: https://damask.mpie.de/release/documentation/processing_tools/index.html#processing-tools can be a good starting point.
There are also some jupyter-notebook examples: https://damask.mpie.de/release/documentation/examples/index.html that can help you understand how to manipulate your input files.

With respect to material parameters: https://damask.mpie.de/release/documentation/usage.html , under the subsection 1. Pre- Processing/ DAMASK Materialpoint model, you can find the `config.tar.xz` files that contains some curated damask material configuration files. That can be helpful too.


Thanks

**Reply by st4rkman** [reply]
Thank you. I'll look into it and try the modifications.

**Reply by st4rkman** [reply]
@sharanroongta Hi, I wanted to ask that lets say if I've made a solver for calculating elastic/plastic deformation behavior (which fullfills my need as per application) and want it to integrate with DAMASK and use the solver along with it so that I can use DAMASK's processing features but using my solver. Can it be done? Can you help me with it?

**Reply by sharanroongta** [reply]
Hi @st4rkman, In principle yes. 
However, all development takes in privately hosted gitlab and you would have to request login credentials and sign the CLA agreement- https://damask.mpie.de/release/development/contribute.html
Would that work for you?

**Reply by st4rkman** [reply]
Thanks @sharanroongta ! That sounds good to me. I shall once consult my supervisor regarding the same then get back to you.
